index.html

```diff
@@ -0,0 +1,462 @@
+              </article>
+              <article>
+                <h4><a href="http://technonjr.org/news/thermax-selects-4-techno-njr-students/" title="Thermax selects 4 Techno NJR Students">Thermax selects 4 Techno NJR Students</a></h4>
+                <small>Congratulations to the selected students: 1. Chetan...</small><hr>
+              </article>
+              <article>
+                <h4><a href="http://technonjr.org/news/startup-venture-wricks-by-ex-students-of-techno-india-njr-incubated-at-techno-india-njr-incubation-center-has-won-the-second-prize-of-canadian-12500/" title="Startup Venture Wricks by ex-students of Techno India NJR incubated at Techno India NJR Incubation Center has won the second prize of Canadian $12500/-">Startup Venture Wricks by ex-students of Techno India NJR incubated at Techno India NJR Incubation Center has won the second prize of Canadian $12500/-</a></h4>
+                <small>Startup Venture Wricks by Lokesh Puri Goswami and Kunjpreet...</small>
+              </article>
+            </div>
+          </div>
+        </div>
+      </div>
+  </div>
+  <div class="row">
+    <div class="col-md-5">
+      <a href="http://technonjr.org/news" class="btn btn-lg btn-secondary">READ MORE</a>
+    </div>
+  </div>
+
+  <div class="container" style="padding-top:2%;">
+    <div class="row">
+      <div class="col-md-5">
+      </div>
+      <div class="col-md-7">
+        <div class="card mb-3" style="min-height:20px; box-shadow:10px 5px 10px #888888">
+          <div class="card-body">
+            <div class="card-text">
+              <h2><b><i>INTERNATIONAL RESEARCH</i></b></h2>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-5">
+      </div>
+      <div class="col-md-7">
+        <div class="card text-center mb-3" style="box-shadow:none; min-height:20px;">
+            <img src="http://technonjr.org/wp-content/uploads/2015/03/website-poster-1024x576.jpg" class="card-image-top" alt="International Research">
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-5">
+      </div>
+      <div class="col-md-7">
+        <a href="http://technonjr.org/department/departments/research/" class="btn btn-lg btn-secondary">READ MORE</a>
+      </div>
+    </div>
+  </div>
+</section>
+
+<!--Departments-->
+
+<section id="departments" style="padding-top:10%;">
+  <div class="container">
+    <div class="title">
+      <h1>DEPARTMENTS</h1>
+      <hr>
+    </div>
+    <div class="row">
+      <div class="col-md-4">
+        <div class="card text-center mb-3">
+          <img class="details card-image-top" src="http://technonjr.org/wp-content/uploads/2014/10/computer-science-550x400.jpg" alt="Computer Science Engineering &#038; Information Technology (CSE &#038; IT)">
+          <div class="image__overlay image__overlay--blur">
```

Comment:
```suggestion
          <img class="details card-image-top" src="" alt="Computer Science Engineering &#038; Information Technology (CSE &#038; IT)">
          <div class="image__overlay image__overlay--blur">
```
Download and use local image here